apps/sms/views/conversation/js/conversation.js

```diff
@@ -2001,12 +2007,28 @@ var ConversationView = {
   },
 
   delete: function conv_delete() {
```

Comment:
__nit:__ I'd rather have something like ```deleteEntireThread``` boolean parameter for this delete method instead, you can easily pass it with ```this.delete.bind(this, true /* deleteEntireThread */)```